specification/cognitiveservices/data-plane/LUIS/Runtime/readme.md

```diff
@@ -66,4 +66,4 @@ Please also specify `--go-sdk-folder=<path to the root directory of your azure-s
 
 ``` yaml $(tag) == 'runtime_2_0' && $(go)
 output-folder: $(go-sdk-folder)/services/cognitiveservices/luis/v2.0/runtime
```

Comment:
During yesterday's meeting @joshgav brought up the sloppy appearance of having `v2.0` and `luis` backwards as compared to the other packages. Would now be a good time to clean these things up?

Reply:
This was intentional, I think we need to include "luis" in the name someplace.  If we don't want it under this directory then I suggest we change the package name to "luisruntime".

Reply:
CC @joshgav

Reply:
Or we could switch it to /services/cognitiveservices/v2.0/luis/runtime.  This might work better actually as there is already a /services/cognitiveservices/v2.0/textanalytics so it would group them under the same v2.0 directory.  I will make this change.